doc/source/conf.py

```diff
@@ -41,6 +41,11 @@
 import sunpy.data
 sunpy.data.download_sample_data(overwrite=False)
 
+# -- Shut up numpy warnings from WCSAxes
+
+import numpy as np
+np.seterr(invalid='ignore')
```

Comment:
the 3.6 switch was for a lot of other warnings (like ones inside pandas), the build wont use astropy master though, I will go hunting for the fix.